sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcReadBenchmark.scala

```diff
@@ -483,6 +484,64 @@ object OrcReadBenchmark {
     }
   }
 
+  def filterPushDownBenchmark(values: Int, width: Int): Unit = {
+    val benchmark = new Benchmark(s"Filter Pushdown", values)
+
+    withTempPath { dir =>
+      withTempTable("t1", "nativeOrcTable", "hiveOrcTable") {
+        import spark.implicits._
+        val selectExpr = (1 to width).map(i => s"CAST(value AS STRING) c$i")
+        val whereExpr = (1 to width).map(i => s"NOT c$i LIKE '%not%exist%'").mkString(" AND ")
```

Comment:
Why not using some simple predicate? Like `col > 5`?

Reply:
Is it important for this config PR?
Let’s focus on the original purpose of this PR.

Reply:
This is kind of the best case for PPD, as the data is sorted. I'm fine with it, but let's add some more cases, at least `==` and `>`. We should follow other benchmarks in this file to make it completed.

Reply:
Ur, @cloud-fan and @gatorsmile .

The best case for PPD is **Spark needs to do lots of processing on the returned rows but ORC reader only returns one stripe with minimal CPU code**.

So, I designed this benchmark in order to show the difference clearly.
1. The push-downed predicate is only `uniqueID = 0` (minimal). We can change that into `uniqueID ==` or `uniqueID >`.
2. `LIKE` predicate is chosed because it's not pushed down and makes Spark do more processing. It's just one of the example of that kind of operation. You can ignore thoses predicates. We can choose some UDFs instead.

Reply:
I mean `LIKE '%not%exist%'` will not be optimized by `LikeSimplification`.

Reply:
oh sorry I missed the `uniqueID` part. So the `like` operation is just to make the difference larger? We don't need to do this, just a simple predicate like `col = 1` or `col < 1`,  to show normally how much PPD improves performance.

Reply:
The goal of this benchmark is not to show the best case of PPD. We just want to see the perf difference of the most common cases.